### DISCUSSION TITLE
当使用腾讯云-上海地区的VPS的IPV6的环境申请ssl时，出现Wait for propagation [timeout错误

### DISCUSSION THREAD

**Discussion by dpy013** [1Panel-dev/1Panel#6669] [Q&A]
### 联系方式

[-email](1650649574@qq.com)

### 1Panel 版本

v1.10.18-lts

### 问题描述

最近从[这里](https://cloud.tencent.com/document/product/1207/104187)申请了腾讯云轻量IPV6的内测，发现使用1p面板无法给域名申请ssl。

### 重现步骤

第一步，在IPV6环境下登陆已安装好的1p面板，
第二步，点开菜单栏的网站/证书，
第三步，填写申请ssl证书的必要信息，例如
Acme 账户
DNS 账户
第四步，执行申请证书的步骤，这时候会卡在Wait for propagation [timeout的错误提示上。

### 期待的正确结果

可以正常的申请ssl

### 相关日志输出

```shell
2024/10/09 19:12:43 开始申请证书,域名 [example.top] 申请方式 [DNS 自动] DNS 账号 [腾讯云] 厂商 [TencentCloud]
2024/10/09 19:12:43 [INFO] [example.top] acme: Obtaining bundled SAN certificate
2024/10/09 19:12:43 [INFO] [example.top] AuthURL: https://acme.zerossl.com/v2/DV90/authz/Uixxxxxxxxxxxxxxxx_xx
2024/10/09 19:12:43 [INFO] [example.top] acme: Could not find solver for: http-01
2024/10/09 19:12:43 [INFO] [example.top] acme: use dns-01 solver
2024/10/09 19:12:43 [INFO] [example.top] acme: Preparing to solve DNS-01
2024/10/09 19:12:46 [INFO] [imydd.top] acme: Trying to solve DNS-01
2024/10/09 19:12:46 [INFO] [example.top] acme: Checking DNS record propagation. [nameservers=8.8.8.8:53,8.8.4.4:53]
2024/10/09 19:12:56 [INFO] Wait for propagation [timeout: 30m0s, interval: 10s]
```

**Comment by zhengkunwang223** [comment]
候选跳过DNS验证试一下

**Comment by dpy013** [comment]
> 候选跳过DNS验证试一下

以下是跳过dns的日志：
2024/10/09 22:01:43 开始申请证书,域名 [example.top] 申请方式 [DNS 自动] DNS 账号 [腾讯云] 厂商 [TencentCloud]
2024/10/09 22:01:43 [INFO] [example.top] acme: Obtaining bundled SAN certificate
2024/10/09 22:01:43 [INFO] [example.top] AuthURL: https://acme.zerossl.com/v2/DV90/authz/66xxxxxxxxxxxxxxx_xxxx
2024/10/09 22:01:43 [INFO] [example.top] acme: Could not find solver for: http-01
2024/10/09 22:01:43 [INFO] [example.top] acme: use dns-01 solver
2024/10/09 22:01:43 [INFO] [example.top] acme: Preparing to solve DNS-01

**Comment by dpy013** [comment]
刚做了测试，绕过dns验证可以申请ssl，
不知道白天行不行。
需具体观察。
